install

```diff
@@ -0,0 +1,593 @@
+    verbose "\n * Setting current Airtime directory as web root..."
+    web_root=${AIRTIMEROOT}/airtime_mvc/public
+elif [ -n "$web_root" ]; then
+    verbose "\n * Creating Apache web root directory..."
+    cp -R ${AIRTIMEROOT}/airtime_mvc/* ${web_root}
+    web_root=${web_root}/public/
+else
+    verbose "\n * Creating default Apache web root directory /usr/share/airtime/..."
+    web_root="/usr/share/airtime"
+    mkdir -p ${web_root}
+    cp -R ${AIRTIMEROOT}/airtime_mvc/* ${web_root}
+    web_root=${web_root}/public/
+fi
+verbose "...Done"
+
+if [ "$apache" = "t" ]; then
+    loud "\n-----------------------------------------------------"
+    loud "                * Installing Apache *                "
+    loud "-----------------------------------------------------"
+
+    loudCmd "apt-get -y --force-yes install apache2 libapache2-mod-php5"
+    set +e
+    apache2 -v | grep "2\.4" > /dev/null
+    apacheversion=$?
+    set -e
+
+    if [ "$apacheversion" != "1" ]; then
+        airtimeconfigfile="airtime.conf"
+    else
+        airtimeconfigfile="airtime"
+    fi
+
+    if [ ! -f /etc/apache2/sites-available/${airtimeconfigfile} ]; then
+        verbose "\n * Creating Apache config for Airtime..."
+        sed -e "s@WEB_ROOT@${web_root}@g" ${SCRIPT_DIR}/installer/apache/airtime-vhost > /etc/apache2/sites-available/${airtimeconfigfile}
+        loudCmd "a2dissite 000-default"
+        loudCmd "a2ensite airtime"
+    else
+        verbose "\nApache config for Airtime already exists, skipping"
+    fi
+fi
+
+if [ "$icecast" = "f" -a ${_i} -eq 1 ]; then
+    echo -e "Install default Airtime Icecast configuration? (Y/n): \c"
+    read IN
+    if [ "$IN" = "y" -o "$IN" = "Y" ]; then
+        icecast="t"
+    fi
+fi
+
+if [ "$icecast" = "t" ]; then
+    loud "\n-----------------------------------------------------"
+    loud "               * Installing Icecast *                "
+    loud "-----------------------------------------------------"
+    
+    loudCmd "DEBIAN_FRONTEND=noninteractive apt-get -y --force-yes install icecast2"
+
+    verbose "\n * Enabling Icecast 2..."
+    sed -i 's/ENABLE=false/ENABLE=true/g' /etc/default/icecast2
+    set +e
+    service icecast2 start
+    set -e
+    verbose "...Done"
+fi
+
+loud "\n-----------------------------------------------------"
+loud "          * Installing Airtime Services *            "
+loud "-----------------------------------------------------"
+
+verbose "\n * Installing necessary binaries..."
+loudCmd "apt-get -y --force-yes install liquidsoap python python-pip"
+loudCmd "pip install setuptools"
+verbose "...Done"
+
+verbose "\n * Creating /usr/lib/airtime..."
+mkdir -p /usr/lib/airtime
+verbose "...Done"
+
+verbose "\n * Creating /run/airtime..."
+mkdir -p /run/airtime
+chmod 755 /run/airtime
+chown -R ${web_user}:${web_user} /run/airtime
+verbose "...Done"
+
+verbose "\n * Copying logging files..."
+cp -R ${AIRTIMEROOT}/python_apps/std_err_override /usr/lib/airtime/
```

Comment:
This needs to be addressed alongside an overhaul of how pypo and media-monitor (possibly also api_client) work